harmonica/_forward/tesseroid.py

```diff
@@ -193,10 +188,9 @@ def tesseroid_gravity(
         kernels[field],
         dtype,
     )
-    result *= GRAVITATIONAL_CONST
-    # Convert to more convenient units
+    # Convert to more convenient units and Invert sign
     if field == "g_z":
-        result *= 1e5  # SI to mGal
+        result *= -1e5  # SI to mGal
```

Comment:
For now, it makes sense that this is a single `if` statement. But since we'll probably add more acceleration components, I think we should split this `if` into two, like you did with the Cartesian forward.

Reply:
> For now, it makes sense that this is a single `if` statement. But since we'll probably add more acceleration components, I think we should split this `if` into two, like you did with the Cartesian forward.

Yup~ That makes sense :)